systemtest/conftest.py

```diff
@@ -16,9 +16,9 @@
 REPO_ROOT = Path(__file__).parent.parent
 sys.path.insert(0, str(REPO_ROOT))
 
-# Set up logging to log at DEBUG level; this is quelled by pytest except when
+# Set up logging to log at INFO level; this is quelled by pytest except when
 # there are errors
-logging.basicConfig(level=logging.DEBUG)
+logging.basicConfig(level=logging.INFO)
```

Comment:
At DEBUG, any failure dumps unbelievably large amounts of output that isn't particularly interesting, so I bumped this to INFO.